front/src/components/VacationsComponents/VacationsPanel.tsx

```diff
@@ -0,0 +1,27 @@
+import ThisMonthVacations from "./ThisMonthVacations.tsx";
+import {useState} from "react";
+
+const VacationsPanel = ({onAddNewVacOpenPopup} : {onAddNewVacOpenPopup: () => void}) => {
+    const [isShowVacations, setIsShowVacationsOpen] = useState<boolean>(false);
+    const [isEditVacations, setIsEditVacations] = useState<boolean>(false);
+
+    return (
+        <div className="flex flex-col gap-4">
+            {isShowVacations && <div className="order-1"> <ThisMonthVacations showVacations={isShowVacations}/> </div>}
```

Comment:
`ThisMonthVacations` is kind very specific. if you want to show the vacations for the next two months you will need a new component?